app/assets/stylesheets/components/_layout.scss

```diff
@@ -6,7 +6,16 @@
   }
 }
 
-.usa-layout-docs-main_content {
+.main-section {
+  @include outer-container();
+  @include padding(null $site-margins-mobile);
+
+  @include media($medium-screen) {
+    @include padding(null $site-margins);
+  }
+
+  max-width: $site-max-width;
```

Comment:
Are there existing markup or class names in the USWDS we could rely on here? Maybe some of the `usa-grid` stuff?

Reply:
I could change all the usa-section things to usa-grid and inherit its CSS. Not sure if that's preferable if we're not using it to layout grid items

Reply:
Good point. Since it's a toss-up, I say go with your changes here. 👍